src/main/kotlin/blackjack/domain/blackjack/BlackJackBuilder.kt

```diff
@@ -0,0 +1,25 @@
+package blackjack.domain.blackjack
+
+import blackjack.domain.card.CardDeck
+import blackjack.domain.card.Cards
+import blackjack.domain.participants.Participants
+import blackjack.domain.participants.ParticipantsBuilder
+
+fun blackJack(block: BlackJackBuilder.() -> Unit): BlackJack {
+    return BlackJackBuilder().apply(block).build()
+}
+
+class BlackJackBuilder {
+    private val cardDeck: CardDeck = CardDeck(Cards.all().shuffled())
+    private lateinit var participants: Participants
+    fun participants(block: ParticipantsBuilder.() -> Unit) {
+        participants = ParticipantsBuilder().apply(block).build()
+    }
+
+    fun initDraw() = participants.all().forEach {
+        it.draw(cardDeck.nextCard())
+        it.draw(cardDeck.nextCard())
+    }
```

Comment:
```suggestion
    fun initDraw() {
        participants.all().forEach {
            it.draw(cardDeck.nextCard())
            it.draw(cardDeck.nextCard())
        }
    }
```
`=` 을 사용하게 되면, 흔히 코드만 봤을 때 마치 리턴값이 있는 것처럼 보여요.
리턴타입이 없는 경우에는 `=` 보다는 중괄호를 사용해보면 어떨까요? (이하 관련 내용 동일)